joschahenningsen/TUM-Live
# Login Page CSS

### Description
Use new start page CSS also for the Login Page. The login logic has stayed the same. Furthermore, remove unused `<head>` imports such as fontawesome.

### Steps for Testing
- 1 Account

1. Log in
2. Reset password (See #1069 for correctness)
3. (Not sure if there is a good way to test the SAML login)

### Screenshots
<div style="display: flex">
<img width="320" alt="Screenshot 2023-07-10 at 14 37 53" src="https://github.com/joschahenningsen/TUM-Live/assets/29633518/ff0916de-274e-48b2-9a31-d713f6986a25">
<img width="320" alt="Screenshot 2023-07-10 at 14 37 43" src="https://github.com/joschahenningsen/TUM-Live/assets/29633518/c20a778a-4163-4782-a40f-6528169340af">
</div>
